utensor_cgen/backend/utensor/_graph_lower.py

```diff
@@ -0,0 +1,23 @@
+from utensor_cgen.backend.base import BackendPart
+from utensor_cgen.utils import class_property
+
+
+class uTensorGraphLower(BackendPart):
+
+  TARGET = 'utensor'
+  PART = 'graph_lower'
+
+  def apply(self, ugraph):
+    handler = getattr(self, 'handle_{}'.format(ugraph.lib_name))
```

Comment:
All backend should be mandated to implement a common uTensor IR handler. Otherwise, we cannot share optimization passes between different front/backend configurations. Of course, additional front/backend handlers can be added, but these are secondary and optional to the uTensor IR.

I'll be opening an issue to add a prefix `uTensor_` to all uTensor operator names (ones that actually gets to JinJa2). Then, start defining the uTensor IR. We have the option to adapt ONNX, TF Lite or our own.

Reply:
I don't think add prefix is a good solution.
I'm thinking a more robust way such as add a legalization phase to the cli.

Graph legalization should be responsible to convert all platform dependent ops to generic IR ops.
And we implement our optimizers, matchers, ...etc against the generic IR graph.
Then in graph lowering phase, we convert generic IR graph to target dependent graph.

However, graph legalization is not included in this PR and hence the graph lowering here is like experimental and requires further refactoring.
I add lowering in this PR is just a temporary workaround on the issue (pytorch, tflite-micro, ...etc) we talk about few days ago.